docs/docsite/rst/porting_guides/porting_guide_core_2.19.rst

```diff
@@ -650,6 +650,31 @@ Values formerly represented by that type will now appear as a tagged ``str`` ins
 Special handling in plugins is no longer required to access the contents of these values.
 
 
+No implicit conversion of non-string keys in returned dictionaries
```

Comment:
This technically applies to any JSON dumping using an Ansible profile-based JSON serializer, not just data returned from modules -- that just happens to be the most common case.

```suggestion
No implicit conversion of non-string dict keys
```